src/sentry/options/defaults.py

```diff
@@ -160,6 +160,12 @@
 # regards to filter responses.
 register("store.lie-about-filter-status", default=False)
 
+# Skip nodestore save when saving an event
+register("store.save-event-skips-nodestore", default=False, flags=FLAG_PRIORITIZE_DISK)
+
+# Check postgres for duplicates before saving an event
+register("store.check-duplicates", default=True, flags=FLAG_PRIORITIZE_DISK)
```

Comment:
```
# If the value is defined on disk, use that and don't attempt to fetch from db.
# This also make the value immutable to changes from web UI.
```

This is what FLAG_PRIORITIZE_DISK does. Is that intended ? (Like not change through the UI)

Reply:
We should typically have this on every option. It doesn’t mean it can’t be changed. It just means if we define it explicitly in our config file, we take an optimized path and don’t check external data stores.

Reply:
thanks